### ISSUE TITLE
Numpy Methods for class docstrings are not supported

### ISSUE THREAD

**Issue by GoogleGenius** [mkdocstrings/python#26]
**Describe the bug**
Using the numpy docs style to specify a ``Methods`` section for class methods is not parsed into a table, unlike ``Attributes``, for example. https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

**To Reproduce**
```py
"""Blah blah blah...

Attributes
----------
closed: bool
    Blah blah blah...

Methods
-------
close()
    Blah blah blah...
"""
```

**Expected behavior**
Docstring is parsed correctly into a table.

**Screenshots**
``Attributes`` is correctly parsed into a table, however, the ``Methods`` is appearing as regular text with no formatting.

<img src="https://user-images.githubusercontent.com/81032623/174517123-f6b9d9dd-426b-4588-a240-193645cc747d.png" height="750" />

**System**
- `mkdocstrings-python` version: 0.7.1
- `Python` version: 3.10.4
- `OS`: Windows

**Additional context**
I am not sure whether the numpy style guide specifically says that type hints can also be used in the ``Methods`` section but noting this here just in case you can as well. 

Would that look like this?
```py
"""
Methods
-------
close() -> None
    Blah blah blah...
"""
```

**Comment by pawamoy** [comment]
Indeed, "Methods" sections are not supported by the underlying parser (Griffe), because such sections do not exist in the Google style, and I wanted to have a common denominator of sections only. I'll mark this as a feature request: I'm open to adding support for Methods sections, but it should be added to both Google and Numpy parsers (Sphinx is probably not concerned, since it has no concept of sections I think?).

**Comment by GoogleGenius** [comment]
One thing about Google though is that I'm not sure whether it's supported as part of its official style guide: https://google.github.io/styleguide/pyguide.html#384-classes

Classes only list out the attributes here.

**Comment by pawamoy** [comment]
This is now implemented, and released as part of mkdocstrings-python 1.5.0 :slightly_smiling_face:
